sa/proto/sa.proto

```diff
@@ -103,11 +103,7 @@ message CountCertificatesByNamesRequest {
 }
 
 message CountByNames {
-  message MapElement {
-          string name = 1;
-          int64 count = 2;
-  }
-  repeated MapElement countByNames = 1;
+  map<string, int64> counts = 1;
```

Comment:
Realizing now that we could do this one in a backwards-compatible way by adding this new field as `= 2`, switching all the code to use it instead, and then removing the old field. That's probably The Right Way to do it, but since we have the atomic deploy anyway for the various Order changes I think this is okay.

Reply:
Oooh, this is an excellent approach to point out though, thanks.